src/views/GovernanceReviewTeam/Actions/index.tsx

```diff
@@ -0,0 +1,193 @@
+      {
+        /* Show form if action is selected */
+        action ? (
+          <Switch>
+            <Route
+              path="/governance-review-team/:systemId/actions/request-edits"
+              render={() => <RequestEdits />}
+            />
+            <Route
+              path="/governance-review-team/:systemId/actions/next-step"
+              render={() => <NextStep />}
+            />
+            <Route
+              path="/governance-review-team/:systemId/actions/decision"
+              render={() => <SubmitDecision />}
+            />
+          </Switch>
+        ) : (
+          /* Select action page */
+          <>
+            <PageHeading
+              data-testid="grt-actions-view"
+              className="margin-top-0 margin-bottom-5"
+            >
+              {t('chooseAction.heading')}
+            </PageHeading>
+
+            <form
+              onSubmit={e => {
+                e.preventDefault();
+                history.push(`actions/${actionRoute}`);
+              }}
+              className="margin-bottom-4"
+            >
+              <ActionContext.Provider
+                value={{
+                  name: 'Available Actions',
+                  onChange: (e: React.ChangeEvent<HTMLInputElement>) => {
+                    setActionRoute(e.target.value);
+                  },
+                  value: actionRoute
+                }}
+              >
+                <RadioGroup className="grt-actions-radio-group grid-row grid-gap-md">
+                  <ActionRadioOption
+                    label={t('chooseAction.requestEdits.title')}
+                    description={t('chooseAction.requestEdits.description')}
+                    accordionText={t('chooseAction.requestEdits.accordion')}
+                    route="request-edits"
+                  />
+                  <ActionRadioOption
+                    label={t('chooseAction.progressToNewStep.title')}
+                    description={t(
+                      'chooseAction.progressToNewStep.description'
+                    )}
+                    accordionText={t(
+                      'chooseAction.progressToNewStep.accordion'
+                    )}
+                    route="next-step"
+                  />
+                  <ActionRadioOption
+                    label={t(`chooseAction.decision${state}.title`, {
+                      context: decisionState
+                    })}
+                    description={t(
+                      `chooseAction.decision${state}.description`,
+                      {
+                        context: decisionState
+                      }
+                    )}
+                    accordionText={t(
+                      `chooseAction.decision${state}.accordion`,
+                      {
+                        context: decisionState
+                      }
+                    )}
+                    route="decision"
+                  />
```

Comment:
I was assuming that all the actions would be dynamic based on [Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?node-id=1%3A25686&mode=dev), but they are hard coded here.  I assumed that this would be handled as a part of this ticket but I may be wrong if there is BE work to be done.  I think it's still possible to account for the dynamic nature of this render without it though.

Reply:
From my understanding, the first two actions should be hard coded but the third decision action is dynamic. I used [i18next context](https://www.i18next.com/translation-function/context) to dynamically get the [decision action text from the translation file](https://github.com/CMSgov/easi-app/blob/7afc2fa2b8e990b32c0abfc882aa2f4da0a71fdd/src/i18n/en-US/action.ts#L20-L51) based on the `state` and `decisionState`.